EIPS/eip-draft_Transferring+ethers+with+privacy+protection+using+precompiled+contracts+for+ring+signature+verification__abbrev.md

```diff
@@ -0,0 +1,78 @@
+## Preamble
+
+    EIP: 
+    Title: Transferring ethers with privacy protection using precompiled contracts for ring signature verification.
+    Author: Jack Lu <jack@wanchain.org>, Demmon Guo <demmon@wanchain.org> and Shi Liu <shi@wanchain.org>
+    Type: Standard Track
+    Category (*only required for Standard Track): Core 
+    Status: Draft
+    Created: 2017-08-21
+    
+
+
+## Simple Summary
+A mechanism is designed to provide privacy protection for transferring ethers using one-time account and precompiled contracts for linkable ring signature verification.
+
+## Abstract
+This EIP suggests implementing a special precompiled contract for ring signature verification and adding one-time account mechanism on Ethereum. This can in turn be combined with *#eip-draft\_Precompiled contracts for ring signature verification on the elliptic curve alt\_bn128.* to provide privacy protection for ether transactions without affecting the in-protocol mechanisms. In each transaction, a one-time account is created for the receiver, of which no one can figure out the very owner without the corresponding scan key. Ring signature is verified by precompiled contracts and helps the receiver transfer ethers to his/her main account from one-time account, which is hidden in a set of one-time accounts to keep untraceable.
+
+## Motivation
+Currently, ether transactions on Ethereum are fully transparent, which makes them unsuitable for several use-cases where users’ privacy is of concern and the link between sender and receiver should not be retrieved by anyone else. #196 and #197 suggest adding precompiled contracts for addition and scalar multiplication on a specific pairing-friendly elliptic curve to verify zkSNARKs in Ethereum smart contract, thus increasing the privacy for users. But it has high computation complexity. Monero solves this problem by using Cryptonote under UTXO model. However, this technology cannot be directly used in Ethereum under account model. Considering all of the above, this EIP forgoes zero knowledge proof and proposes to implement a special precompiled contract for ring signature verification and add one-time account mechanism on Ethereum to provide privacy protection for ether transactions. Note that, with this smart contract, users can choose regular transactions or transactions with privacy protection depending on their needs for privacy and anonymity.
```

Comment:
Account abstraction discussed in #678 hopes to provide a mechanism to address these problems, I hope that this precompiled contract will be a useful building block to solve problems like this.